debuglog-plugin/src/test/kotlin/com/bnorm/debug/log/TestUtils.kt

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (C) 2020 Brian Norman
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bnorm.debug.log
+
+import com.strobel.assembler.InputTypeLoader
+import com.strobel.assembler.metadata.ArrayTypeLoader
+import com.strobel.assembler.metadata.CompositeTypeLoader
+import com.strobel.assembler.metadata.ITypeLoader
+import com.strobel.decompiler.Decompiler
+import com.strobel.decompiler.DecompilerSettings
+import com.strobel.decompiler.PlainTextOutput
+import com.tschuchort.compiletesting.KotlinCompilation
+import com.tschuchort.compiletesting.SourceFile
+import java.io.ByteArrayOutputStream
```

Comment:
Weirdly `ktlint` was not passing until I moved the `java` imports above `org`. https://github.com/pinterest/ktlint/issues/527